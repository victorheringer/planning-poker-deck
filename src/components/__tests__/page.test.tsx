import { render } from "@testing-library/react";
import Page from "../page";

describe("Logo", () => {
  it("should render correctly", () => {
    const tree = render(<Page />).asFragment();
    expect(tree).toMatchSnapshot();
  });
});
