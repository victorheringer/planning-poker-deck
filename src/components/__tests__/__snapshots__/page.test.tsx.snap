// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo should render correctly 1`] = `
<DocumentFragment>
  <div
    class="sc-bdvvtL fgRsHJ"
  />
</DocumentFragment>
`;
